File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Iconv.java
package com.onevgo.functions;

import java.io.UnsupportedEncodingException;

public class Iconv {
    public static String iconv(String inCharset, String outCharset, String str) {
        try {
            return new String(str.getBytes(inCharset), outCharset);
        } catch (UnsupportedEncodingException e) {
            e.printStackTrace();
        }
        return str;
    }

    public static void main(String[] args) {
        String text = "This is the Euro symbol '€'.";
        System.out.println(text);
        System.out.println(iconv("utf-8", "iso-8859-1", text));
    }
}

File: 01_Language/01_Functions/python/iconv.py
# coding: utf-8

def iconv(in_charset, out_charset, s):
    return s.encode(in_charset).decode(out_charset)


if __name__ == '__main__':
    text = "This is the Euro symbol '€'."
    print(text)
    print(iconv("utf-8", "iso-8859-1", text))
